notebooks/langchain-api-client.ipynb

```diff
@@ -0,0 +1,453 @@
+{
```

Comment:
<ol><li>How are we defining the model that would be used by the API, is it hardcoded for now, if so can we mention that?</li><li>Minor nitpick (feel free to ignore): we could add the credentials.env workflow (load LLM_API_URI from the cred file) here to keep it consistent with the open-ai notebooks.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
> 1. How are we defining the model that would be used by the API, is it hardcoded for now, if so can we mention that?

It is out of control from the client side: the model is whatever the API app is serving at the time - there is no way (in our current serving app) to select a model via the API.

I will add a short mention about this.

>     2. Minor nitpick (feel free to ignore): we could add the credentials.env workflow (load LLM_API_URI from the cred file) here to keep it consistent with the open-ai notebooks.

Fair enough - updating.